lib/core/src/recover/recoverer.rs

```diff
@@ -213,11 +212,21 @@ impl Recoverer {
                             log::warn!("Could not apply recovered data for incoming Chain swap {swap_id}: recovery data not found");
                             continue;
                         };
+                        if chain_swap.payer_amount_sat != recovered_data.btc_user_lockup_amount_sat
+                        {
+                            chain_swap.actual_payer_amount_sat =
```

Comment:
Is there a reason you see not to populate the actual_payer_amount_sat regardless if it is different than the payer_amount_sat?

Reply:
My idea was to use the presence or absence of the `actual_payer_amount_sat` to tell if this was an over/underpaid swap. However, always populating it and then checking if it differs from `payer_amount_sat` also works. I don't have a preference for either approach.

Reply:
For me it makes sense to populate it anyway. For consistency.

Reply:
Applied in [0cf8e2b](https://github.com/breez/breez-sdk-liquid/pull/615/commits/0cf8e2b9ad592553c2683c0990b62aa838e5aab3)